### PR TITLE
:sparkles: Source Jira: certification updates: docs, project_permission_schemes stream, issues expand made as default

### PR DESCRIPTION
**Pull request by darynaishchenko**
## What
- **docs**:  Removed inapp.md, added Pull Request to incremental streams, removed issue expand
- **project_permission_schemes** stream: updated unittest, fixed mocked data(it were duplicates with previous implementation, because of duplicated project ids)
- **spec** - removed optional "issue expand", because it relates to implementation only, not user

### CODE REVIEW

**Comment by airbyte-oss-build-runner** [comment]
##  source-jira test report (commit [`64616fb070`](https://github.com/airbytehq/airbyte/pull/30688/commits/64616fb070d67a81ea790ab30c9922c50ca22db8)) - ❌

⏲️  Total pipeline duration: 08mn19s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Connector package install                                          | ✅        |
| Build source-jira docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                         | ✅        |
| Integration tests                                                  | ✅        |
| Acceptance tests                                                   | ❌        |
| Code format checks                                                 | ✅        |
| Validate airbyte-integrations/connectors/source-jira/metadata.yaml | ✅        |
| Connector version semver check                                     | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-jira_adding-tests/1695377077/64616fb070d67a81ea790ab30c9922c50ca22db8/source-jira/0.7.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:64616fb070d67a81ea790ab30c9922c50ca22db8)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-jira test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-jira test report (commit [`033613cb0f`](https://github.com/airbytehq/airbyte/pull/30688/commits/033613cb0f6d56fff72c1b22c7798b70f07216c9)) - ✅

⏲️  Total pipeline duration: 08mn28s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Connector package install                                          | ✅        |
| Build source-jira docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                         | ✅        |
| Integration tests                                                  | ✅        |
| Acceptance tests                                                   | ✅        |
| Code format checks                                                 | ✅        |
| Validate airbyte-integrations/connectors/source-jira/metadata.yaml | ✅        |
| Connector version semver check                                     | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-jira_adding-tests/1695390625/033613cb0f6d56fff72c1b22c7798b70f07216c9/source-jira/0.7.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:033613cb0f6d56fff72c1b22c7798b70f07216c9)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-jira test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-jira test report (commit [`2f63c830c2`](https://github.com/airbytehq/airbyte/pull/30688/commits/2f63c830c2facc214dc59996dde86079dbfc74d9)) - ❌

⏲️  Total pipeline duration: 09mn21s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Connector package install                                          | ✅        |
| Build source-jira docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                         | ✅        |
| Integration tests                                                  | ✅        |
| Acceptance tests                                                   | ❌        |
| Code format checks                                                 | ✅        |
| Validate airbyte-integrations/connectors/source-jira/metadata.yaml | ✅        |
| Connector version semver check                                     | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-jira_adding-tests/1695722567/2f63c830c2facc214dc59996dde86079dbfc74d9/source-jira/0.7.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:2f63c830c2facc214dc59996dde86079dbfc74d9)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-jira test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-jira test report (commit [`4811418921`](https://github.com/airbytehq/airbyte/pull/30688/commits/4811418921d7e37aca04881dee092d19a5b0a883)) - ❌

⏲️  Total pipeline duration: 09mn39s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Connector package install                                          | ✅        |
| Build source-jira docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                         | ✅        |
| Integration tests                                                  | ✅        |
| Acceptance tests                                                   | ❌        |
| Code format checks                                                 | ✅        |
| Validate airbyte-integrations/connectors/source-jira/metadata.yaml | ✅        |
| Connector version semver check                                     | ✅        |
| Connector version increment check                                  | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-jira_adding-tests/1695724272/4811418921d7e37aca04881dee092d19a5b0a883/source-jira/0.7.3/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:4811418921d7e37aca04881dee092d19a5b0a883)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-jira test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-jira test report (commit [`7aea4d4311`](https://github.com/airbytehq/airbyte/pull/30688/commits/7aea4d4311b31f169d7166e2f709ecd51ec04575)) - ✅

⏲️  Total pipeline duration: 09mn16s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Connector package install                                          | ✅        |
| Build source-jira docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                         | ✅        |
| Integration tests                                                  | ✅        |
| Acceptance tests                                                   | ✅        |
| Code format checks                                                 | ✅        |
| Validate airbyte-integrations/connectors/source-jira/metadata.yaml | ✅        |
| Connector version semver check                                     | ✅        |
| Connector version increment check                                  | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-jira_adding-tests/1695726501/7aea4d4311b31f169d7166e2f709ecd51ec04575/source-jira/0.7.3/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:7aea4d4311b31f169d7166e2f709ecd51ec04575)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-jira test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-jira test report (commit [`b57d0a1a74`](https://github.com/airbytehq/airbyte/pull/30688/commits/b57d0a1a7469769e0458d435683bd432ff4ca90d)) - ✅

⏲️  Total pipeline duration: 10mn52s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Connector package install                                          | ✅        |
| Build source-jira docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                         | ✅        |
| Integration tests                                                  | ✅        |
| Acceptance tests                                                   | ✅        |
| Code format checks                                                 | ✅        |
| Validate airbyte-integrations/connectors/source-jira/metadata.yaml | ✅        |
| Connector version semver check                                     | ✅        |
| Connector version increment check                                  | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-jira_adding-tests/1695743919/b57d0a1a7469769e0458d435683bd432ff4ca90d/source-jira/0.8.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:b57d0a1a7469769e0458d435683bd432ff4ca90d)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-jira test
```

**Review by davydov-d — APPROVED** [review]

**Review by artem1205 — CHANGES_REQUESTED** [review]
Please remove `MigrateIssueExpandProperties` from `main.py`

**Comment by airbyte-oss-build-runner** [comment]
##  source-jira test report (commit [`828f047df5`](https://github.com/airbytehq/airbyte/pull/30688/commits/828f047df52b7893d7be44d6d0fea2e65d7e652c)) - ❌

⏲️  Total pipeline duration: 10mn37s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Connector package install                                          | ✅        |
| Build source-jira docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                         | ✅        |
| Integration tests                                                  | ✅        |
| Acceptance tests                                                   | ✅        |
| Code format checks                                                 | ❌        |
| Validate airbyte-integrations/connectors/source-jira/metadata.yaml | ✅        |
| Connector version semver check                                     | ✅        |
| Connector version increment check                                  | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-jira_adding-tests/1695821021/828f047df52b7893d7be44d6d0fea2e65d7e652c/source-jira/0.8.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:828f047df52b7893d7be44d6d0fea2e65d7e652c)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-jira test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-jira test report (commit [`9d8690a315`](https://github.com/airbytehq/airbyte/pull/30688/commits/9d8690a315fde1a2ec3543a3f9c0c75789f52323)) - ✅

⏲️  Total pipeline duration: 10mn15s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Connector package install                                          | ✅        |
| Build source-jira docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                         | ✅        |
| Integration tests                                                  | ✅        |
| Acceptance tests                                                   | ✅        |
| Code format checks                                                 | ✅        |
| Validate airbyte-integrations/connectors/source-jira/metadata.yaml | ✅        |
| Connector version semver check                                     | ✅        |
| Connector version increment check                                  | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-jira_adding-tests/1695823692/9d8690a315fde1a2ec3543a3f9c0c75789f52323/source-jira/0.9.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:9d8690a315fde1a2ec3543a3f9c0c75789f52323)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-jira test
```

**Review by artem1205 — APPROVED** [review]

File: docs/integrations/sources/jira.md [hunk 1]
```diff
@@ -25,8 +25,7 @@ This page contains the setup guide and reference information for the Jira source
...
-9. Fill in **Expand Issues stream** with values from the suggested list. See `expand` in [Query parameters](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issue-search/#api-rest-api-3-search-get-request-Query%20parameters) for more info.
+8. Enter the **Start Date (Optional)** from which you'd like to replicate data for Jira in the format YYYY-MM-DDTHH:MM:SSZ. All data generated after this date will be replicated, or leave it empty if you want to replicate all data. Note that it will be used only in the following streams: Board Issues, Issue Comments, Issue Properties, Issue RemoteLinks, Issue Votes, Issue Watchers, Issue Worklogs, Issues, Pull Requests, Sprint Issues. For other streams it will replicate all data.
```
Comment (brianjlai) [inline docs/integrations/sources/jira.md:28]: nit: Fix `Issue RemoteLinks` to `Issue Remote Links`

**Review by brianjlai — APPROVED** [review]
✅

**Comment by airbyte-oss-build-runner** [comment]
##  source-jira test report (commit [`f89bf42cb7`](https://github.com/airbytehq/airbyte/pull/30688/commits/f89bf42cb7e2851d03dc6034daaf79cc6c0cd629)) - ✅

⏲️  Total pipeline duration: 09mn19s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Connector package install                                          | ✅        |
| Build source-jira docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                         | ✅        |
| Integration tests                                                  | ✅        |
| Acceptance tests                                                   | ✅        |
| Code format checks                                                 | ✅        |
| Validate airbyte-integrations/connectors/source-jira/metadata.yaml | ✅        |
| Connector version semver check                                     | ✅        |
| Connector version increment check                                  | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-jira_adding-tests/1695887476/f89bf42cb7e2851d03dc6034daaf79cc6c0cd629/source-jira/0.9.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:f89bf42cb7e2851d03dc6034daaf79cc6c0cd629)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-jira test
```

**Pull request merged**